src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -1074,6 +1088,8 @@ private sealed class BatchingDocumentCollection
             /// </summary>
             private readonly List<DocumentId> _documentsRemovedInBatch = new List<DocumentId>();
 
+            private readonly ImmutableArray<ImmutableArray<string>>.Builder _multipleOrderedFilePathsInBatch = ImmutableArray.CreateBuilder<ImmutableArray<string>>();
```

Comment:
If this is only ever an `ImmutableArray<T>.Builder`, why not just use List like everything else?

Reply:
Because I saw a builder being used here: "private readonly ImmutableArray<DocumentInfo>.Builder _documentsAddedInBatch " - perhaps that should have been a list too?